php/Terminus/Site.php

```diff
@@ -292,25 +292,54 @@ public function import($url, $element) {
   }
 
   /**
-   * get site instrument
+   * Changes payment instrument of given site
+   *
+   * @params [string] $uuid UUID of new payment instrument
+   * @return [Workflow] $workflow Workflow object for the request
    */
-  public function instrument($org=null) {
-    $path = 'instrument';
-    $method = 'GET';
-    $options = null;
-    if ($org) {
-      $path = 'workflows';
-      $data = array(
-          'type'    => 'transfer_site_payment_to_organization',
-          'params'  => array(
-              'to_organization_id' => $org,
-          ),
-        );
-      $options = array( 'body' => json_encode($data) , 'headers'=>array('Content-type'=>'application/json') );
-      $method = 'POST';
+  public function changeInstrument($uuid = 0) {
```

Comment:
In this case, we'd write less code by exposing 2 methods than by also writing a third methods that wraps the other two. 

This also just comes from my general experience working on the internal Pantheon API is that methods that seem symmetric (eg add and remove) are not. So I prefer to follow a "least surprise" principle and write a 1-to-1 interface rather than try to compose more complex functions (like "change").